sklearn/feature_selection/tests/test_rfe.py

```diff
@@ -328,3 +329,21 @@ def test_rfe_cv_n_jobs():
     rfecv.fit(X, y)
     assert_array_almost_equal(rfecv.ranking_, rfecv_ranking)
     assert_array_almost_equal(rfecv.grid_scores_, rfecv_grid_scores)
+
+
+def test_rfe_cv_groups():
+    # test cv with groups
```

Comment:
you can remove the comment since this is already implied by the name of the function